src/components/schemas/statement_mappings.yaml

```diff
@@ -0,0 +1,81 @@
+suggest_out:
+  type: object
+  additionalProperties: False
+  properties:
+    id:
```

Comment:
`created_at` and `updated_at` are missing

Reply:
Not needed for this api sir, as this is a suggestion api.